copy/copy.go

```diff
@@ -133,6 +133,11 @@ type Options struct {
 	// Invalid when copying a non-multi-architecture image. That will probably
 	// change in the future.
 	EnsureCompressionVariantsExist []OptionCompressionVariant
+
```

Comment:
A super-tiny nit: Without this empty line we could structure `Options` a bit more into “paragraphs” with related options.

Reply:
> Ah nevermind if we error out user will be forced to provide a `--compression`

Yes, that’s the idea.  I don’t think “use a CLI flag to force a compression, but rely on a config file to specify which one” is something users will really need… if they _care_ to force a compression, I’m fairly comfortable with having them type out which one they care about in the CLI as well.

Also note that `containers.conf` is not handled inside c/image — c/image just sees `copy.Options` that make little sense.

Admittedly if Buildah or Podman always set `CompressionFormat` value (e.g. from a `containers.conf` — I didn’t check if that’s the case), that detection of a “force but not say what” wouldn’t work I’m afraid, but that’s not something c/image can fix locally.